src/api/status/src/js/octokit.js

```diff
@@ -0,0 +1,31 @@
+const { Octokit } = require('@octokit/core');
+const { retry } = require('@octokit/plugin-retry');
+const { throttling } = require('@octokit/plugin-throttling');
+
+const MyOctokit = Octokit.plugin(retry, throttling);
+const octokit = new MyOctokit({
+  auth: process.env.GITHUB_TOKEN,
```

Comment:
This is good.  We'll need some other changes to use it in docker.  I'll file an issue.

Reply:
Filed #2799